qds_sdk/cloud/oracle_bmc_cloud.py

```diff
@@ -200,4 +211,90 @@ def create_parser(self, argparser):
         storage_config.add_argument("--storage-api-private-rsa-key",
                                     dest="storage_api_private_rsa_key",
                                     default=None,
-                                    help="storage api private rsa key for oracle cluster")
\ No newline at end of file
+                                    help="storage api private rsa key for oracle cluster")
+
+
+    def get_results(self, cmd, fp=sys.stdout, storage_credentials=None, r={}, delim=None, qlog=None, include_header=None):
+        import oraclebmc
+        private_key = storage_credentials.get('storage_config').get('api_private_rsa_key')
+        private_key_file_path = "/tmp/" + uuid.uuid4().hex
+        with open(private_key_file_path, 'w+') as private_key_file:
+            private_key_file.write(private_key)
+        os.chmod(private_key_file_path, 0o744)
+        oracle_config = {"user": storage_credentials.get('storage_config').get('user_id'),
+                         "key_file": private_key_file_path,
+                         "fingerprint": storage_credentials.get('storage_config').get('key_finger_print'),
+                         "tenancy": storage_credentials.get('storage_config').get('tenant_id'),
+                         "region": "us-phoenix-1",  # just now added us-phoenix-1 , have to change api to get this param
+                         "pass_phrase": None,
+                         "additional_user_agent": "",
+                         "log_requests": False
+                         }
+        object_storage = oraclebmc.object_storage.ObjectStorageClient(oracle_config)
+        log.info("Starting download from result locations: [%s]" % ",".join(r['result_location']))
+        num_result_dir = Command.find(cmd.id).num_result_dir
+        for oracle_path in r['result_location']:
+            _download_to_local_oracle(object_storage, oracle_path, fp, delim=delim,
+                                      skip_data_avail_check=isinstance(self, PrestoCommand))
+
+
+def _download_to_local_oracle(storage_conn, oracle_path, fp, delim=None, skip_data_avail_check=False):
```

Comment:
skip_data_avail_check is not being used. We can check whether the complete data is available in the same manner as we do for aws?